shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -470,6 +470,13 @@ - (BOOL)handleKeyEvent:(NSEvent*)event {
   if (!_shown) {
     return NO;
   }
+
+  // NSTextInputContext sometimes randomly deactivates itself without calling
```

Comment:
nit: Consider removing the word 'randomly' here since it's almost certainly deterministic.